extensions/pathfactory/pathfactory-content-track.schema.json

```diff
@@ -0,0 +1,236 @@
+            },
+            "pathfactory:isHeaderEnabled": {
+              "required": [],
+              "type": "boolean",
+              "description": "Boolean variable to check if header is enabled in PathFactory",
+              "title": "Is Header Enabled",
+              "meta:xdmType": "boolean"
+            },
+            "pathfactory:isCookieMessageEnabled": {
+              "required": [],
+              "type": "boolean",
+              "description": "Boolean variable to check if cookie message is enabled in PathFactory",
+              "title": "Is Cookie Message Enabled",
+              "meta:xdmType": "boolean"
+            },
+            "pathfactory:isFlowEnabled": {
+              "required": [],
+              "type": "boolean",
+              "description": "Boolean variable to check if flow is enabled in PathFactory",
+              "title": "Is Flow Enabled",
+              "meta:xdmType": "boolean"
+            },
+            "pathfactory:isSignpostEnabled": {
+              "required": [],
+              "type": "boolean",
+              "description": "Boolean variable to check if signpost is enabled in PathFactory",
+              "title": "Is Signpost Enabled",
+              "meta:xdmType": "boolean"
+            },
+            "pathfactory:isTopicSidebarEnabled": {
+              "required": [],
+              "type": "boolean",
+              "description": "Boolean variable to check if topic sidebar is enabled in PathFactory",
+              "title": "is topic sidebar enabled",
+              "meta:xdmType": "boolean"
+            },
+            "pathfactory:isLikeEnabled": {
+              "required": [],
+              "type": "boolean",
+              "description": "Boolean variable to check if like is enabled in PathFactory",
+              "title": "Is Like Enabled",
+              "meta:xdmType": "boolean"
+            },
+            "pathfactory:isEndEnabled": {
+              "required": [],
+              "type": "boolean",
+              "description": "Boolean variable to check if end is enabled in PathFactory",
+              "title": "Is End Enabled",
+              "meta:xdmType": "boolean"
+            },
+            "pathfactory:targetContentOrder": {
+              "required": [],
+              "type": "string",
+              "description": "Target Content Order",
+              "title": "Target Content Order",
+              "meta:xdmType": "string"
+            },
+            "pathfactory:updatedAtUtc": {
+              "required": [],
+              "format": "date-time",
+              "type": "string",
+              "description": "Updated date in UTC format",
+              "title": "Updated Date (UTC)",
+              "meta:xdmType": "date-time"
+            }
+          },
+          "meta:xdmType": "object"
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "#/definitions/pfContentTrackFields"
+    }
+  ],
+  "meta:status": "stable"
```

Comment:
Please meake the meta:status : experimental. We can mark that status stable only after testing on staging